.github/labeler.yml

```
@@ -1,25 +1,16 @@
 # https://github.com/actions/labeler#common-examples
-# Adapted from https://github.com/rapidsai/cudf/blob/main/.github/CODEOWNERS
+# Adapted from https://github.com/rapidsai/cuxfilter/blob/main/.github/CODEOWNERS
 # Labels culled from https://github.com/rapidsai/cuxfilter/labels
 
 Python:
- - python/**
- - notebooks/**
+ - 'python/**'
+ - 'notebooks/**'
 
-libcuxfilter:
- - cpp/**
+gpuCI:
+   - 'ci/**'
 
-CMake:
- - **/CMakeLists.txt
- - **/cmake/**
-
-Java:
- - java/**
-
-Ops:
- - .github/**
- - /ci/**
- - conda/**
- - **/Dockerfile
- - **/.dockerignore
- - docker/**
+conda:
+   - 'conda/**'
+ 
+ doc:
```

```suggestion
doc:
```
Fixing indentation issue